data/formats-data.js

```diff
@@ -4474,7 +4481,7 @@ exports.BattleFormatsData = {
 		tier: "NU",
 	},
 	patrat: {
-		randomBattleMoves: ["swordsdance", "batonpass", "substitute", "hypnosis", "return", "superfang"],
+		randomBattleMoves: ["swordsdance", "batonpass", "return", "crunch"],
```

Comment:
keep super fang and hypnosis for stall breaking

Reply:
you want me to stall break with a patrat? id rather just sd if i want to hurt things and have a chance to not be totally useless after